Model/Config.php

```diff
@@ -497,7 +498,11 @@ public function getMethodLogo()
      */
     public function isInContextCheckout()
     {
-        return false;  //iframe checking is disable until zip checkout js support iframe for all browser
+        $currentCurrencyCode = $this->_storeManager->getStore()->getCurrentCurrencyCode();
+        if ($currentCurrencyCode !== CommonUtil::CURRENCY_AUD){
```

Comment:
I haven't done too much (if any haha) PHP coding in my time, so I'm curious to know why sometimes you use `!==` and sometimes `!=` (like [here](https://github.com/zipMoney/zip.magento2/pull/34/files#diff-dc7ff48cacc1bd8ebab238beae4ef558589f996757cee7e2db0afd9219d6941fR247))?